base/array.jl

```diff
@@ -1596,14 +1596,14 @@ cat(n::Integer, x::Integer...) = reshape([x...], (ntuple(x->1, n-1)..., length(x
 """
     findnext(A, i::Integer)
 
-Find the next linear index >= `i` of a non-zero element of `A`, or `0` if not found.
+Find the next linear index >= `i` of a true element of `A`, or `0` if not found.
```

Comment:
Maybe add backticks around `true` (here and in other docstring) since "a true element" may not be completely obvious when you have no idea what the function does.